addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -249,7 +249,7 @@
             <field name="context">{
                 'search_default_week':1,
                 'search_default_groupby_employee':1,
-                'search_default_groupby_project':1,
+                'search_default_groupby_project':2,
```

Comment:
It doesn't change anything right ? 

Or I'm missing something ?

Reply:
Hello @tivisse 
It will change the sequence of the applying group-by.
Currently, the group by in 'All timesheet' is project > employee
Here, we changes the group by sequence so it will now apply as employee > Project

Thanks